packages/salesforcedx-vscode-apex/src/messages/i18n.ts

```diff
@@ -75,5 +75,8 @@ export const messages = {
     '%s points to a missing folder. For information on how to setup the Salesforce Apex extension, see [Set Your Java Version](%s).',
   wrong_java_version_text:
     'An unsupported Java version was detected. Download and install [Java 8](https://java.com/en/download/) or [Java 11](https://www.oracle.com/technetwork/java/javase/downloads/jdk11-downloads-5066655.html) to run the extensions. For more information, see [Set Your Java Version](%s).',
-  force_apex_test_suite_build_text: 'SFDX: Build Apex Test Suite'
+  force_apex_test_suite_build_text: 'SFDX: Build Apex Test Suite',
+  unable_to_locate_editor: 'You can run this command only on a source file.',
+  unable_to_locate_document: 'You can run this command only on a source file.',
```

Comment:
New strings.  I'm working with Sonal on the verbage to use.

Reply:
I heads back from Sonal, and the strings to be used are:
"You can run this command only on a source file."
"You can run this command only on an Anonymous Apex file, or an Apex test file."

Reply:
Update 2/2:  Since the command has been merged to also run debug log files, the error message has been updated to, "You can only run this command with Anonymous Apex files, Apex Test files, or Apex Debug Log files."  Confirmed & approved by @sbudhirajadoc